### ISSUE TITLE
Measure mask area

### ISSUE THREAD

**Issue by halordbo** [dbolya/yolact#233]
Hi, is there possibly a way to record the pixel area of the masks per unit time?

**Comment by ljjyxz123** [comment]
> Hi, is there possibly a way to record the pixel area of the masks per unit time?

What do you mean by per unit time?

Do you want to calculate the area of the masks in instance segmentation in real-time?

**Comment by halordbo** [comment]
Yes, and also log the area of the masks for every frame I guess?

**Comment by dbolya** [comment]
You'll have to implement that yourself, but masks are created here: https://github.com/dbolya/yolact/blob/02bde378e98cd7e2831639ff782c1f5a5046771a/eval.py#L155

All eval options use the same code to generate and display masks, so implement your logging code here and then use any of the video options to run.

**Comment by shihaojie123** [comment]
> Hi, is there possibly a way to record the pixel area of the masks per unit time?

Brother, how is it, did you calculate the mask area?
